extensions/azcli/src/typings/az-ext.d.ts

```diff
@@ -392,6 +392,17 @@ declare module 'az-ext' {
 					usek8s?: boolean,
 					// Additional arguments
 					additionalEnvVars?: AdditionalEnvVars
+				): Promise<AzOutput<void>>,
+				upgrade(
+					desiredVersion: string,
```

Comment:
Usually the non-standard args have been put into a separate "args" object to more easily separate them out from the rest.

Reply:
Got it, put the direct and indirect params under args